core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -401,12 +387,17 @@ extern "C" fn new_confirm_action(n_args: usize, args: *const Obj, kwargs: *mut M
         let mut page = if hold {
             ButtonPage::new(paragraphs, theme::BG).with_hold()?
         } else {
-            ButtonPage::new(paragraphs, theme::BG).with_cancel_confirm(verb_cancel, verb)
+            ButtonPage::new(paragraphs, theme::BG)
+                .with_cancel_confirm(verb_cancel.map(|c| c.into()), verb.map(|c| c.into()))
```

Comment:
instead: `verb_cancel: Option<TString> = ....try_into_option()?;

(again the same thing about replacing all `x: StrBuffer` with `x: TString` should apply here)

Reply:
should be fixed in f358885376b67f824950f81bd6c83670e423ed18